stdlib/public/Concurrency/TaskGroup.swift

```diff
@@ -220,21 +215,15 @@ public struct TaskGroup<ChildTaskResult: Sendable> {
     }
   }
 
-  /// Add a child task to the group.
-  ///
-  /// ### Error handling
-  /// Operations are allowed to `throw`, in which case the `try await next()`
-  /// invocation corresponding to the failed task will re-throw the given task.
-  ///
-  /// The `add` function will never (re-)throw errors from the `operation`.
-  /// Instead, the corresponding `next()` call will throw the error when necessary.
+  /// Adds a child task to the group.
   ///
   /// - Parameters:
-  ///   - overridingPriority: override priority of the operation task
-  ///   - operation: operation to execute and add to the group
-  /// - Returns:
-  ///   - `true` if the operation was added to the group successfully,
-  ///     `false` otherwise (e.g. because the group `isCancelled`)
+  ///   - overridingPriority: The priority of the operation task.
+  ///     Omit this parameter or pass `.unspecified`
+  ///     to set the child task's priority to the priority of the group.
+  ///   - operation: The operation to execute as part of the task group.
+  /// - Returns: `true` if the operation was added to the group successfully;
+  ///   otherwise; `false`.
```

Comment:
Sorry this is my bad -- need to remove the `returns` -- the `spawn` does not return anything now.

Only `spawnUnlessCancelled` does return `bool`